Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -14,16 +14,31 @@ public abstract class Combatant : MonoBehaviour
     public int CurrentHealthPoints;
     public int MaxShieldPoints;
     public int CurrentShieldPoints;
-    private readonly float HealthBarPosition = 125f;
     protected CombatSystem CombatSystem;
     [SerializeField]
     GameObject HealthBarPrefab;
     [SerializeField]
+    GameObject ShieldBarPrefab;
+    [SerializeField]
     GameObject HealthBarUIPanel;
     [SerializeField]
     GameObject Shield;
 
     private GameObject HealthBar;
+    private HealthBar RedBar;
+    private Text HealthText;
+
+    private bool IsShieldActive;
+    private GameObject ShieldBar;
+    private HealthBar BlueBar;
+    private Text ShieldText;
+
+    private const float HealthBarYOffset = 1.45f;
```

Comment:
Call it Height instead?

Reply:
This is more of a scaler, I'll update the name